app/serializers/message.js

```diff
@@ -0,0 +1,18 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  attrs: {
+    conversations: { serialize: true }
+  },
+
+  serialize(snapshot) {
+    let json = this._super(...arguments);
+
+    // support saving child conversations alongside record
+    if (!json.id) {
+      json.included = snapshot.hasMany('conversations').map((c) => c.serialize());
+    }
```

Comment:
This is how we include the conversation alongside the message in our payload when creating a message.

Apparently, ember data does not do this automatically and provides no configuration in order to do it, so I had to provide a manual implementation. It's according to spec, as far as I know, but the sad reality is, this whole process involves custom code both API and client side.

Reply:
This looks about right!  Is there an equivalent `conversation_part` on the ember side?